jenkins/slave-base/Dockerfile.rhel7

```diff
@@ -91,8 +93,13 @@ RUN if [ -z $SNYK_DISTRIBUTION_URL ] ; then echo 'Skipping snyk installation!' ;
   && snyk --version \
   && echo 'Snyk installation completed!'; fi
 
-
 # set java proxy var
 COPY set_java_proxy.sh /tmp/set_java_proxy.sh
 RUN chmod 777 /tmp/set_java_proxy.sh
 RUN . /tmp/set_java_proxy.sh && echo $JAVA_OPTS
+
+# add skopeo
+RUN yum install -y http://mirror.centos.org/centos/7/extras/x86_64/Packages/ostree-${OSTREE_VERSION}.el7.x86_64.rpm
+RUN yum install -y http://mirror.centos.org/centos/7/extras/x86_64/Packages/containers-common-${SKOPEO_VERSION}.el7.centos.x86_64.rpm
+RUN yum install -y http://mirror.centos.org/centos/7/extras/x86_64/Packages/skopeo-${SKOPEO_VERSION}.el7.centos.x86_64.rpm 
+RUN skopeo --version
```

Comment:
It would be nice to put all 4 instructions into one `RUN` instruction to reduce the amount of layers (which is still a best practice even though not super-crucial anymore, see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers).

Also, I assume there are no RedHat provided RPMs?

Reply:
@michaelsauter  - I have not found RHEL ones - and relied on what @tbugfinder reported to work ... still cool?

Reply:
Yes, as also the other binaries we install are not RedHat provided, so I don't see a general issue using the CentOS provided ones ...

Reply:
My understanding is that RHEL7.7 and RHEL8 should include it within additional repos.

I was working with Jenkins-Slave - Images based on RHEL7.6.

Reply:
@tbugfinder - i did not find it in any - can you point, otherwise as we close ODS 2 .. I'll merge

Reply:
I didn't find it either.
Thx.